src/utils/dbFunctions.js

```diff
@@ -0,0 +1,79 @@
+import User from "@/models/User";
+import Showtime from "@/models/Showtime";
+import Booking from "@/models/Booking";
+import Review from "@/models/Review";
+import Movie from "@/models/Movie";
+
+// USERS
+
+export async function addUser(data) {
+  return await User.create(data);
+}
+
+export async function getUsers() {
+  return await User.find();
+}
+
+export async function getOneUser(id) {
```

Comment:
we can maybe have two methods for this? 
Instead of (id), we can maybe have username and email.
To easier see if the user is already registered, when creating an account or logging in? :)

Reply:
For sure, I'll add those!